app/javascript/components/ui/datepicker/datepicker-component.js

```diff
@@ -27,6 +34,97 @@ class Datepicker extends PureComponent {
           }}
           focused={this.state.focused}
           onFocusChange={({ focused }) => this.setState({ focused })}
+          navPrev={
+            <div className="c-navigation-button c-button-left">
```

Comment:
There are a couple of points here:
- this is a button, so for accessibility we should use the correct html tags `<button>` when declaring buttons.
- this is messy and complicated. Luckily for you we have a couple of components that handle all of this for you already with styles and themes. https://github.com/Vizzuality/gfw/tree/develop/app/javascript/components/ui/button and https://github.com/Vizzuality/gfw/tree/develop/app/javascript/components/ui/icon

Take a look and implement. You can search the project for these components to easily find examples.